common/test/acceptance/tests/lms/test_lms_problems.py

```diff
@@ -161,3 +161,35 @@ def test_demand_hint(self):
                 {'event': {u'hint_index': 0, u'hint_len': 2, u'hint_text': u'demand-hint1'}}
             ],
             actual_events)
+
+
+class ProblemPartialCredit(ProblemsTest):
+    """
+    Makes sure that the partial credit is appearing properly.
+    """
+    def get_problem(self):
```

Comment:
Ah, great. Thanks @Colin-Fredericks.  I wonder if that's missing the `@abstractmethod` decorator, but that can be covered separately...not a concern for this PR.